src/presets/azure.ts

```diff
@@ -41,14 +41,31 @@ async function writeRoutes(nitro: Nitro) {
     }
   }
 
+  // Attempt to load custom config
+  let customConfig = {};
+  try {
+    customConfig = JSON.parse(
```

Comment:
This can fail silently if JSON.parse fails (invalid json). 
Maybe we should actually throw an error if JSON.parse fails ? 
One way to do it would be to check if the file exist first.
 
`const customConfig = exists ? JSON.parse() : {}`